script.js

```diff
@@ -1,129 +1,173 @@
-/*
- * jQuery v1.9.1 included
- */
+document.addEventListener('DOMContentLoaded', function() {
 
-$(document).ready(function() {
+  function matches (element, selector) {
```

Comment:
since this is only used in case `closest` is not defined, would it make sense to have this logic inlined in the `closest` function?